Add unit tests for condition9

EdwardNorberg opened the pull request:
Two unit tests, one should resolve to true, one should resolve to false.

#62

hampfh committed:
feat: Add new utility functions to point class

hampfh committed:
feat: Add condition6

hampfh committed:
Merge branch 'main' of github.com:kth-cdate-courses/DD2480 into hampus/feat/8-add-condition6

hampfh committed:
fix: Merge conflict

hampfh committed:
fix: Add back behavior from this branch

hampfh committed:
Merge branch 'main' of github.com:kth-cdate-courses/DD2480 into hampus/feat/8-add-condition6

Commit:
feat: Condition 8 method #10

Commit:
feat: Condition 13 method #14

Commit:
feat: Create method for PUM #18

hampfh committed:
fix: Fix bug in (#8) related indexing of the end point

Commit:
fix: Create method for PUM #18

Cleaner switch case for assignment of b

Commit:
fix: Missing ; #18

paulinegarelli committed:
Merge branch 'main' into pauline/feat/18-pum-computation

hampfh committed:
Merge branch 'main' into hampus/feat/8-add-condition6

Commit:
feat: Condition 5 method #7

Commit:
fix: Missing ) #7

Commit:
feat: Condition 11 method #12

EdwardNorberg committed:
Add unit tests for condition9

Two unit tests, one should resolve to true, one should resolve to false.

EdwardNorberg commented:
I'm trying to figure out how there is a merge conflict when the branch I'm merging into should have the same lines of code that cause the issue

EdwardNorberg committed:
Change back condition8 to condition7

Because of apperent merge conflic. Will see

EdwardNorberg committed:
Merge branch 'edward/feat/61-unit-tests-condition8' into edward/feat/62-unit-test-condition9

Commit:
feat: quadrant method for Point #6

Add method to determine the quadrant in which a point lies

Commit:
feat: quadrantRepartition method for Point #6

Add method to determine if a set of points are divided in more than a certain number of quadrants

hampfh reviewed (changes requested):
I have done a dry run of this on paper and to me it seems like both tests should return false, in the first test we create an angle between the points that is pi > pi + episilon = `pi > 3pi/2` which will yield false, the second test will also yield false but there it is expect which is fine

Commit:
feat: Condition 4 method #6

paulinegarelli committed:
Merge branch 'main' into pauline/feat/6-condition4

hampfh committed:
fix: Fix bug related to k2

hampfh committed:
Merge pull request #34 from kth-cdate-courses/hampus/feat/8-add-condition6

Condition 6 implementation

paulinegarelli committed:
Merge branch 'main' into pauline/feat/10-condition8

Commit:
fix: change number of point condition #10

Checking that we have at least 5 points and not 3 in condition 8

Commit:
Merge branch 'pauline/feat/10-condition8' of github.com:kth-cdate-courses/DD2480 into pauline/feat/10-condition8

paulinegarelli committed:
Merge pull request #43 from kth-cdate-courses/pauline/feat/10-condition8

Condition 8 method #10

Commit:
fix: non-strict comparison to radius 2 #14

In condition 13, the points are considered to be contained in a circle even if they are on the line, so the comparison should be not strict.

paulinegarelli committed:
Merge branch 'main' into pauline/feat/14-condition13

paulinegarelli committed:
Merge pull request #44 from kth-cdate-courses/pauline/feat/14-condition13

Condition 13 method #14

paulinegarelli committed:
Merge branch 'main' into pauline/feat/18-pum-computation

Commit:
fix: missing ; #18

paulinegarelli committed:
Merge pull request #45 from kth-cdate-courses/pauline/feat/18-pum-computation

Create method for PUM #18

paulinegarelli committed:
Merge pull request #70 from kth-cdate-courses/pauline/feat/7-condition5

Pauline/feat/7 condition5

paulinegarelli committed:
Merge pull request #71 from kth-cdate-courses/pauline/feat/12-condition11

Condition 11 method #12

hampfh committed:
Merge pull request #72 from kth-cdate-courses/edward/feat/47-unit-tests-condition1

Add unit tests for condition1

hampfh committed:
Merge branch 'main' of github.com:kth-cdate-courses/DD2480 into pauline/feat/6-condition4

hampfh committed:
Merge pull request #84 from kth-cdate-courses/pauline/feat/6-condition4

Pauline/feat/6 condition4

EdwardNorberg committed:
Merge branch 'main' of https://github.com/kth-cdate-courses/DD2480 into edward/feat/62-unit-test-condition9

EdwardNorberg committed:
Fixed according to comment

Test that should yield true gave wrong answer, new points should give
correct outcome.

EdwardNorberg committed:
Merge branch 'edward/feat/62-unit-test-condition9' of https://github.com/kth-cdate-courses/DD2480 into edward/feat/62-unit-test-condition9
